migrations/20231212184028_indexes.sql

```diff
@@ -0,0 +1,9 @@
+-- Redundant with finished_at
+ALTER TABLE
+    job_queue DROP COLUMN completed_at;
```

Comment:
nit(naming): Should we keep `completed_at` and drop `finished_at`? Any thoughts into which name fits better?

Reply:
I'm kind of indifferent, but wondering if there's any reason to prefer one over the other.

Reply:
As it's _currently_ used, `finished_at` feels slightly more correct. Because we set it even when we put a row back in for a `retry`:

https://github.com/PostHog/rusty-hook/blob/126908fd4f57b5b85bb6b618b4229a17e67d892b/hook-common/src/pgqueue.rs#L285-L316

To me, `completed` really means we're done (`completed` or `failed`) where it seems like we're using `finished_at` as more like a `last_attempt_finished_at` (maybe that over verbose name would be better?)

Reply:
Yeah, I agree `completed_at` feels more final and implies it was with `completed` status. We could have one timestamp per status, i.e. `completed_at`, `failed_at`, `retried_at`, but that gets redundant quickly and hard to maintain if we ever add another status.

Reply:
It could also be an array, only the final timestamp could be `completed` or `failed` anyway, anything before the final basically has to be "attempt that failed". I guess all that gets us is knowledge of when failed attempts happened, I don't know how useful that is. You're right that we could add more statuses anyway, though.